### ISSUE TITLE
Improve focus outline for keyboard navigation in bookmarks content card (Firefox)

### ISSUE THREAD

**Issue by sairina** [learningequality/kolibri#9192]
## Observed behavior
When in Bookmarks, the keyboard navigation skips over the content card and jumps to the info icon and close button on the content card.
![2022-03-14 15 16 57](https://user-images.githubusercontent.com/13563002/158270201-56139fd4-f5bd-457b-8a86-86d1ac480079.gif)

## Errors and logs
None

## Expected behavior
The first focused element should be the content card

## User-facing consequences
Difficult for user to click on content card via keyboard navigation 

## Steps to reproduce
1. Log in as learner
2. Add some content to bookmark
3. Go to the bookmarks tab (with your mouse)
4. Once in bookmarks, use keyboard to navigate
5. Notice that the first focusable element is the icon button.

## Context
Kolibri 0.15
Firefox

**Comment by marcellamaki** [comment]
I think the skipping over is another possible mac settings issue. With the settings updated, the problem becomes the actual focus outline (which, as I recall, was discussed with @radinamatic pre-0.15, as it does appear, but doesn't actually surround the entire card). I would propose rescoping this issue to improve the focus outline, if the settings can be confirmed to be the problem with the outline not appearing. 

![bookmarks-navigation-firefox.gif](https://images.zenhubusercontent.com/6047f6e691103a78d3ad0ace/75a7b2b2-b99d-4539-9361-e572eadebebe)

**Comment by marcellamaki** [comment]
cc @radinamatic -- to double check before updating

**Comment by pcenov** [comment]
@marcellamaki yes, I confirm that the issue reported by @sairina was caused by the Mac settings option not being enabled. After enabling it the focus outline appears correctly only in Chrome, and as observed by you it doesn't fully surround the cards in both Firefox and Safari so you are correct to point out that this issue could be rescoped to address that problem.

**Comment by rtibbles** [comment]
Fixed in #9409
